x-pack/plugins/file_upload/public/util/file_parser.js

```diff
@@ -63,15 +63,17 @@ export const fileHandler = async ({
           default:
             for (const feature of batch.data) {
               // TODO: Give feedback on which features failed
-              if ((!feature.geometry || !feature.geometry.type) && !boolGeometryErrs) {
-                boolGeometryErrs = true;
-                errors.push(
-                  new Error(
-                    i18n.translate('xpack.fileUpload.fileParser.featuresOmitted', {
-                      defaultMessage: 'Some features without geometry omitted',
-                    })
-                  )
-                );
+              if (!feature.geometry || !feature.geometry.type) {
+                if (!boolGeometryErrs) {
+                  boolGeometryErrs = true;
+                  errors.push(
+                    new Error(
```

Comment:
It's up to you, but I wonder if we can keep a count of how many features were missing geometry so we can report that back to the user?

Reply:
I agree that it's a good add but would prefer not to in this PR. I've updated [the error tracking issue](https://github.com/elastic/kibana/issues/48194) with this idea and a link back to this comment for context!